example-jobs/checkmarx-security-job-example.groovy

```diff
@@ -8,7 +8,7 @@ new CheckmarxSecurityJobBuilder(
         serverUrl: "http://sample-serverUrl",
         username: "sample-username",
         password: "sample-password",
-        groupId: "${groupId}",
+        groupId: "",
         scanRepo: "https://github.com/OrlandoSoto/ckan-browser-tests",
```

Comment:
because the variable is not defined in the example file, and consequently it errors. It's not taking it out of the builder itself, just the example.

An alternate fix would be to define groupId in the example job, but I don't think that adds anything.